WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/accountmismatch/AccountMismatchRepository.kt

```diff
@@ -99,31 +100,21 @@ class AccountMismatchRepository @Inject constructor(
         data class ConnectedToDifferentAccount(val wpcomEmail: String) : JetpackConnectionStatus
     }
 
-    private suspend fun discoverXMlRPCAddress(siteUrl: String): Result<String> = suspendCancellableCoroutine { cont ->
-        val listener = object : Any() {
-            @Subscribe(threadMode = MAIN)
-            @Suppress("unused")
-            fun onDiscoverySucceeded(event: OnDiscoveryResponse) {
-                dispatcher.unregister(this)
-                if (event.isError) {
-                    WooLog.w(WooLog.T.LOGIN, "XMLRPC Discovery failed, error: ${event.error}")
-                    cont.resume(Result.failure(OnChangedException(event.error, event.error.name)))
-                } else {
-                    WooLog.d(
-                        tag = WooLog.T.LOGIN,
-                        message = "XMLRPC Discovery succeeded, xmrlpc endpoint: ${event.xmlRpcEndpoint}"
-                    )
-                    cont.resume(Result.success(event.xmlRpcEndpoint))
-                }
-            }
-        }
-
+    private suspend fun discoverXMlRPCAddress(siteUrl: String): Result<String> {
```

Comment:
Really small nitpick:
```suggestion
    private suspend fun discoverXMLRPCAddress(siteUrl: String): Result<String> {
```

Reply:
Sorry, I may be blind 😄, what the suggestion is changing?

Reply:
The l -> L 😄

Reply:
I'm definitely going blind 😅